Stadistic/src/utils/ScannerData.java

```diff
@@ -8,8 +8,15 @@ public class ScannerData {
     public static Trial askForData() {
         Scanner scanner = new Scanner(System.in);
 
-        System.out.print("Ingresar nombre: ");
-        Integer idTrial = scanner.nextInt();
+        System.out.println(" ");
+        System.out.println("INGRESO DE PROCESO - SIGA LAS INSTRUCCIONES");
+        System.out.println("============================================");
+        System.out.println(" ");
+
+        //TODO: I'm not sure if the 19 line is correct
+
+        System.out.print("Ingresar número de proceso: ");
+        Integer idTrial = Integer.valueOf(scanner.nextLine());
```

Comment:
I decide to follow the suggestion of IDE about this line, but I'm not sure if is correct to take the string and convert it to an integer, maybe there is a direct option to do this. 🤔 @Katalinais